src/styles/blocks/card.scss

```
@@ -0,0 +1,101 @@
+.main__container {
+  width: 944px;
+  display: flex;
+  padding: 50px 40px;
+  align-items: center;
+  flex-wrap: wrap;
+  margin: 0 auto;
+}
+
+.card {
+  @extend %font;
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+  margin: 0 48px 46px 0;
+
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  &__photo {
+    background-image: url("../images/imac.jpeg");
+    background-repeat: no-repeat;
+    background-size: 160px 134px;
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+    align-self: center;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $color-black;
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    font-weight: 400;
+    font-size: 10px;
+    color: $color-gray;
+    margin-bottom: 17px;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    font-size: 10px;
+    color: $color-black;
+    margin-bottom: 23px;
+  }
+
+  &__price {
+    font-size: 12px;
+    color: $color-gray;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px;
+
+    &-item {
+      font-weight: 700;
+      font-size: 16px;
+      color: $color-black;
+    }
+  }
+
+  &__link {
+    width: 166px;
+    height: 40px;
+    background: $color-blue;
+    border-radius: 5px;
+
+    font-family: "Roboto", sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 14px;
+    text-transform: uppercase;
+    text-decoration: none;
+    color: white;
+
+    display: flex;
+    text-align: center;
+    justify-content: center;
+    align-items: center;
+
+    &:hover {
+      background: white;
+      border: 1px solid $color-blue;
```

Don't add new border to the element on hover. Add default transparent border of the same width, and change its color on :hover
```suggestion
      borde-color: $color-blue;
```